packages/frontend/src/pages/settings/settings-page.tsx

```diff
@@ -31,22 +35,24 @@ const Settings: React.FC = () => {
         }}
         variant="permanent"
       >
-        <List>
-          <ListItemButton
-            selected={settingsPageSelected === 'Item 1'}
-            onClick={() => handleItemClick('Item 1')}
-            sx={{ '&.Mui-selected': { backgroundColor: 'primary.secondary' } }}
-          >
-            <ListItemText primary="Інтеграція Monobank" />
-          </ListItemButton>
-          <ListItemButton
-            selected={settingsPageSelected === 'Item 2'}
-            onClick={() => handleItemClick('Item 2')}
-            sx={{ '&.Mui-selected': { backgroundColor: 'primary.secondary' } }}
-          >
-            <ListItemText primary="Спільний доступ" />
-          </ListItemButton>
-        </List>
+        <ListItemButton
+          sx={{ '&.Mui-selected': { backgroundColor: 'primary.secondary' } }}
+          component={Link}
+          to="integration"
+          selected={location.pathname === '/settings/integration'}
+          onClick={() => handleItemClick('integration')}
```

Comment:
we can put here just call of `navigate` instead of handling local/global state + useEffect

Reply:
I had an idea to remember which part of the settings the user was in. So that the user, when switching to the example in Home, returning to the settings, would get to the same place where he was